projects/storefrontlib/shared/components/generic-link/generic-link.component.ts

```diff
@@ -91,6 +103,19 @@ export class GenericLinkComponent implements OnChanges {
     return this.routeParts.fragment ?? undefined;
   }
 
+  onClick(event: UIEvent): void {
+    event.preventDefault();
+    if (this.isExternalUrl() || this.target === '_blank') {
+      window.open(this.url as string, this.target ?? '');
```

Comment:
It feels a bit intrusive to overwrite the native on-click behavior of `<a>` elements. It should be done with caution. I'm wondering if we break other native behaviors of links as a side effect. For instance:
When I click a link with a pressed `Command` button - I'd expect it would open in new tab.